content/docs/Demo/Accessingdemo.md

```diff
@@ -24,31 +24,17 @@ Conforma can be used on Windows, macOS, and Linux desktop systems, as well as on
 The link to the demo is here: [https://demo.conforma.nz/login](https://demo.conforma.nz/login).
```

Comment:
Just FYI -- the proper link should be [https://demo.conforma.nz/register](https://demo.conforma.nz/register)

Saves a click and reduces possible confusion if you can go straight to the registration form.